### ISSUE TITLE
move:complete is emitted by servoWrite when degrees === last.degrees

### ISSUE THREAD

**Issue by dtex** [rwaldron/johnny-five#829]
```
var servo = new five.Servo(11); // Servo is at 90 by default
servo.to(180); // move:complete is not emitted. Expected.
servo.to(180); // move:complete is emitted. Not expected.
```

Also with the upcoming move to animation() for servo.to() we get:
```
var servo = new five.Servo(11); // Servo is at 90 by default
servo.to(90, 1000); // move:complete is emitted twice. Not expected.
```
I believe that if we are not going to emit move:complete when calling servo.to without a duration, then it should not be emitted just because degrees === last.degrees. In the event that an animation is called, animation() will run the callback that emits the event after the requested period of time has passed.

I think we should remove:
https://github.com/rwaldron/johnny-five/blob/master/lib/servo.js#L29
and
https://github.com/rwaldron/johnny-five/blob/master/lib/servo.js#L105

If we don't remove those two lines, then we should always emit move:complete when calling servo.to() but we've had conversations about why that would be a bad idea (the event will be emitted while the servo is still moving).

**Comment by rwaldron** [comment]
I'm ok with this

**Comment by dtex** [comment]
Will include this change with my animation() wrapping servo.to  PR. That's just waiting on #828 which has a fix pending (thank you for that BTW).

**Comment by rwaldron** [comment]
No problem, but we still need to address that breakage :\

**Comment by dtex** [comment]
Which breakage do you mean? #827 won't affect this. Or did you mean this is a breaking change because it alters previous behavior?

**Comment by rwaldron** [comment]
The breakage I pinged you on [here](https://github.com/rwaldron/johnny-five/commit/2fd7f9caccd5d404f597453c8d153c4cc48ef51c#commitcomment-11754585) and [here](https://github.com/rwaldron/johnny-five/pull/833#issuecomment-113266582)

**Comment by Resseguie** [comment]
Hmm, I somehow didn't see that ping either. I'll try to have a look and see if I have any useful input.

**Comment by dtex** [comment]
Included in #858

**Comment by dtex** [comment]
Forgot to close this.
